packages/manager/modules/bm-server-components/src/netboot/template.html

```diff
@@ -0,0 +1,253 @@
+<div class="container-fluid">
+    <oui-back-button on-click="$ctrl.goBack()">
+        <span data-translate="server_configuration_netboot_title"></span>
+    </oui-back-button>
+
+    <p
+        data-ng-bind="'server_configuration_netboot_current' | translate:{ t0: $ctrl.server.name }"
+    ></p>
+    <p data-ng-bind="$ctrl.server.netbootLabel"></p>
+
+    <oui-stepper data-on-finish="$ctrl.setNetboot()">
+        <!-- Configure the OS -->
+        <oui-step-form
+            id="configurationForm"
+            name="configurationForm"
```

Comment:
```suggestion
            data-id="configurationForm"
            data-name="configurationForm"
```

Reply:
to be compliant with the rest

Reply:
data- is not prefixed for original HTML attributes like id, class, name etc